mne/io/tests/test_meas_info.py

```diff
@@ -228,6 +228,23 @@ def test_read_write_info():
     assert m1 == m2
 
 
+@pytest.mark.skip(reason="XXX TODO this is a regression to investigate")
```

Comment:
My plan is to address this in a subsequent PR (it is actually a BUG)

Reply:
in case you don't do this immediately please explain the pb.

Reply:
This does not fail in master. (I thought it did) So it is something I've introduced. Needs fixing.

Reply:
Actually is just a rounding issue.
```py
def test_anonymization_annotation_regression():
    from numpy.testing import assert_array_almost_equal
    # Fake some subject data
    raw = read_raw_fif(raw_fname)
    expected_onset = [0, 1]
    raw.set_annotations(Annotations(onset=expected_onset,
                                    duration=[1, 1],
                                    description='dummy',
                                    orig_time=None))
    raw.info['subject_info'] = dict(id=1, his_id='foobar', last_name='bar',
                                    first_name='bar', birthday=(1987, 4, 8),
                                    sex=0, hand=1)

    raw.anonymize()
    assert_array_almost_equal(raw.annotations.onset, expected_onset, decimal=7)
```

```sh

 mne/io/tests/test_meas_info.py ✓                                                                                        100% ██████████
====================================================== slowest 20 test durations =======================================================
0.02s call     mne/io/tests/test_meas_info.py::test_anonymization_annotation_regression
0.00s teardown mne/io/tests/test_meas_info.py::test_anonymization_annotation_regression
0.00s setup    mne/io/tests/test_meas_info.py::test_anonymization_annotation_regression

Results (0.20s):
       1 passed
```

So I'll remove the entire thing. It makes no sense.